ts/session/messages/outgoing/content/link/DeviceLinkGrantMessage.ts

```diff
@@ -0,0 +1,60 @@
+import { SignalService } from '../../../../../protobuf';
+import { DeviceLinkRequestMessage } from './DeviceLinkRequestMessage';
+import { LokiProfile } from '../../../../../types/Message';
+
+interface DeviceLinkGrantMessageParams {
```

Comment:
you can do

`interface DeviceLinkGrantMessageParams extends DeviceLinkMessageParams` and avoid duplication

Reply:
Applied for all types of `params`